src/components/toolbar/toolbar.html

```diff
@@ -0,0 +1,19 @@
+<header class="toolbar mdc-toolbar mdc-elevation--z4">
+  <div class="toolbar__container mdc-toolbar__row">
+    <section class="toolbar__container-item mdc-toolbar__section mdc-toolbar__section--align-start">
+      <a href="#" class="material-icons mdc-toolbar__menu-icon menu">menu</a>
```

Comment:
is menu a separate block? maybe it would be better to make it as an element of toolbar block?